centinel/primitives/dnslib.py

```diff
@@ -15,7 +15,7 @@ def get_ips(host, nameserver=None, record="A"):
     return lookup_domain(host, nameservers=nameservers, rtype=record)
 
 
-def lookup_domain(domain, nameservers=[], rtype="A", timeout=10):
+def lookup_domain(domain, nameservers=[], rtype="A", timeout=2):
```

Comment:
We already weren't going to have one clean pcap-> the capture starts before we run the experiment. Also, a single pcap file will be better for storage-> if you store a lot of data, then larger files are better. I don't think we are talking about a large enough pcap for upload problems to be significant either.